bash-preexec.sh

```diff
@@ -43,7 +43,7 @@ fi
 
 # We only support Bash 3.1+.
 # Note: BASH_VERSINFO is first available in Bash-2.0.
-if [[ -z "${BASH_VERSINFO-}" || BASH_VERSINFO[0] -lt 3 || BASH_VERSINFO[0] -eq 3 && BASH_VERSINFO[1] -lt 1 ]]; then
```

Comment:
I'd need to dig through the revision history to be sure, but I suspect this intentionally doesn't use arithmetic expansions. Either way, it would be better to make this change separately from this PR.

Reply:
Thanks! I have introduced the version check in #136. At that time, I just used the single conditional command `[[ ... ]]` because it looked simpler to me compare to combining `[[ ... ]] || (( ... ))`. This time, I have checked the updated version check works as expected with Bash 1.14.7, 2.0, 3.0, and 3.1. Bash 1.14.7 doesn't have BASH_VERSINFO, so it is rejected by `[[ -z "${BASH_VERSINFO-}" ]]`. Bash 2.0+ has an arithmetic command `(( ... ))` and seems to work as expected.

Anyway, I'd separate the PR!

Reply:
I dropped the commit and created a separate PR at #142. Thank you.